packages/shared/tests/normalizeRoutePath.spec.ts

```diff
@@ -204,42 +204,3 @@ describe('should normalize clean paths correctly', () => {
     }),
   )
 })
-
-describe('should normalize paths with query correctly', () => {
```

Comment:
So we are not keeping hash & queries anymore in `normalizeRoutePath`. Any potential breakings?